systemd/grive-sync.sh

```diff
@@ -15,7 +15,7 @@ cd ~
 
 ### ARGUMENT PARSING ###
 SCRIPT="${0}"
-DIRECTORY=$(systemd-escape --unescape "$2")
+DIRECTORY=$(systemd-escape --unescape -- "$2")
 
 if [[ -z "$DIRECTORY" ]] || [[ ! -d "$DIRECTORY" ]] ; then
 	echo "Need a directory name in the current users home directory as second argument. Aborting."
```

Comment:
If this anables directories all over the place, then this line (and the stuff in the readme) can probably be changed :-)